### ISSUE TITLE
Bug: cannot extend any more

### ISSUE THREAD

**Issue by khaledosama999** [json-schema-faker/json-schema-faker#662]
jsf.extend is not a function, I can no longer extend stuff like `faker` or `chance`

**Comment by miladezzat** [comment]
@pateketrueke

**Comment by miladezzat** [comment]
It solved when I changed 

```js
const  jsf = require('json-schema-faker');
```

To :

```js
const  jsf = require('json-schema-faker').default;
```

**Comment by pateketrueke** [comment]
Yeah, I recently started using esbuild to bundle the library, I think we could add an entry-file to avoid having to use the default export may be?

**Comment by pateketrueke** [comment]
Update: I hope [esbuild can handle UMD format](https://github.com/evanw/esbuild/pull/1331) soon, is just matter of time!

**Comment by pateketrueke** [comment]
Hi @khaledosama999 can you elaborate more no this? Thank you!
